### PR TITLE
Make readme more current, and improve usage section

### PR DESCRIPTION
**Pull request by Xaeroxe**
Updates the readme to reflect engine progress, and improves visibility on the Linux developers portion.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ Read a summary of what happened this past week at [*This Week in Amethyst*][tw].
 
 ## Vision
 
-Amethyst aims to be a fast, [data-oriented](https://www.amethyst.rs/book/master/html/glossary.html#data-driven-design), and data-driven game engine suitable
+Amethyst is a fast, [data-oriented](https://www.amethyst.rs/book/master/html/glossary.html#data-driven-design), and data-driven game engine suitable
```
Comment (omni-viral) [inline README.md]: Do we have benchmarks already?

Comment (Xaeroxe) [reply README.md]: I suppose we don't, but then again neither does any other game engine I've seen.

Comment (torkleyy) [reply README.md]: But still, how can you be sure it actually is already?

File: README.md [hunk 2]
```diff
@@ -65,12 +65,14 @@ does not aim to be API-compatible with it in any way. Some goals include:
 
 ## Usage
 
+**Linux developers: the libasound2-dev package will be required to compile amethyst.**
```
Comment (torkleyy) [inline README.md]: Should be "Amethyst"

**Comment by LucioFranco** [comment]
Looks good!

bors r+

**Comment by Xaeroxe** [comment]
bors r-

I made a mistake and didn't fix the A.

**Comment by Xaeroxe** [comment]
bors r=@LucioFranco

Fixed.

**Pull request merged**